File: fontes/pascal/micro/micro01.pas
program Micro01;
    
var 
	cel, far: REAL;
begin
    writeln('Tabela de conversão: Celsius -> Fahrenheit');
    writeln('Digite a temperatura em Celsius: ');
    readln(cel);
    far:=(9*cel+160)/5;
    writeln('A nova temperatura é: ',far,' F');
end.

File: fontes/c/micro/micro01.c
#include<stdio.h>

int main()
{
 float cel;
 float far;

 printf("Tabela de conversão: Celsius -> Fahrenheit\n");
 printf("Digite a temperatura em Celsius: ");
 scanf("%f", &cel);
 far = ((9*cel+160)/(5));
 printf("A nova temperatura é: %f F \n",far);
 return 0;
}